### ISSUE TITLE
Navigation spinner

### ISSUE THREAD

**Issue by davisagli** [plone/plone-react#182]
Since navigation happens via api requests, the browser doesn't show a spinner while pages load. I find this makes the site feel nonresponsive (did my click actually do anything?) even though it's actually faster. I think we should add our own spinner somewhere on the page (in place of the logo the toolbar?)

**Comment by tisto** [comment]
@davisagli I fully agree. I changed the title, hope you are ok with that. It shouldn't be too hard to hook some animation into react router.

**Comment by tahamukhtar20** [comment]
does this issue still exist or it's fixed?
